apps/web/src/components/workflow_v2/StepEditorComponent.tsx

```diff
@@ -27,6 +27,8 @@ export const WorkflowsStepEditor = ({
   onControlsSave?: () => void;
   isSavingControls?: boolean;
 }) => {
+  const errorData = error && 'response' in error ? error?.response?.data : error;
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** We should already have this sort of logic in place somewhere, and if not, we should extract it to its own centralized implementation (in a helper function or something)‏